pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -37,23 +37,44 @@ const (
 	defaultOverCommitFactor = 1.2
 )
 
+const (
+	// cpuOverCommitFactor is resource overCommit factor for cpu
```

Comment:
Use `overcommit-factor.cpu`, `overcommit-factor.memory`. If there are other scalar resource, use `overcommit-factor.ephemeral-storage`,  `overcommit-factor.pods`, etc.

so the key can be cpu/memory/ephemeral-storage/pods, even nvidia.com/gpu.

Reply:
I understand. This change is reasonable.

Reply:
I want to change it to: "overcommit-factor.cpu", "overcommit-factor.memory" "overcommit-factor.ephemeral-storage" and "overcommit-factor.other". The advantage of this change is that I think the number of other pods should not be increased. In addition, other external resources are not allowed to be increased in most scenarios, so we only leave "overcommit-factor.other" for users to set.

Reply:
You haven't get the idea. please think more about extendable.